src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -181,6 +181,10 @@ public function load(array $configs, ContainerBuilder $container)
             $this->registerTemplatingConfiguration($config['templating'], $container, $loader);
         }
 
+        if ($container->hasDefinition('twig')) {
```

Comment:
the container will *never* have a Twig service here, as extensions can only see services they define themselves.